### DISCUSSION TITLE
NPM not showing in webUI all the proxy hosts

### DISCUSSION THREAD

**Discussion by denispollini** [NginxProxyManager/nginx-proxy-manager#3559] [Q&A]
Hi Everyone in NPM WebUI in Proxy Hosts section i see some proxy hosts but not all, if i go in this path i have this proxy host file configuration 12 item in tolal
![immagine](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/81813145/61fac454-e2ed-49b9-aa74-4e42802a1242)
But in WebUI i see only 6 item:
![immagine](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/81813145/004e219e-5073-475d-9acc-8dc4df9da1a3)

**Comment by lastsamurai26** [comment]
> Hi Everyone in NPM WebUI in Proxy Hosts section i see some proxy hosts but not all, if i go in this path i have this proxy host file configuration 12 item in tolal ![immagine](https://private-user-images.githubusercontent.com/81813145/305656422-61fac454-e2ed-49b9-aa74-4e42802a1242.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzMjc4OTYsIm5iZiI6MTcwODMyNzU5NiwicGF0aCI6Ii84MTgxMzE0NS8zMDU2NTY0MjItNjFmYWM0NTQtZTJlZC00OWI5LWFhNzQtNGU0MjgwMmExMjQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDA3MjYzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkZWVmMDBiYWI1NDBiYmFlZDZiOWJlODU5MzFhMmI0OWFjMzI1ZmQ2ODNmMzFkNDFiMzk4YWJmMjgxYWIyMzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1GjoTvMmizZ-QqoMk302HAStjhyHl68DwyyD4xSOwt0) But in WebUI i see only 6 item: ![immagine](https://private-user-images.githubusercontent.com/81813145/305656488-004e219e-5073-475d-9acc-8dc4df9da1a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzMjc4OTYsIm5iZiI6MTcwODMyNzU5NiwicGF0aCI6Ii84MTgxMzE0NS8zMDU2NTY0ODgtMDA0ZTIxOWUtNTA3My00NzVkLTlhY2MtOGRjNGRmOWRhMWEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDA3MjYzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyZTA1MTI0ZmEyMzBjZWU2NzY1YjM0N2NhNTdlMDliZTc2MTA0Y2IwOGY4ZGZjNDNkYTc4NGE5MDFhODZlZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uBHAnHcuRql3Tw8dD2JrizxmZ2FD2jwk81QEEjojQ_c)

do you have more than one user ?

**Reply by denispollini** [reply]
Hi
No only one administrator account.
